client/src/main/java/sigblockchain/projectred/client/Account.java

```diff
@@ -0,0 +1,64 @@
+package sigblockchain.projectred.client;
+
+import java.math.BigInteger;
+
+public class Account {
+	private String WalletAddress;
+	private BigInteger Balance;
+	private BigInteger StateNonce;
+
+	public Account(String walletAddress, BigInteger balance, BigInteger stateNonce) {
+		validateWalletAddress(walletAddress);
+
+		WalletAddress = walletAddress;
+		Balance = balance;
```

Comment:
When assigning instance variables, it should be of the form `this.variable = value`.